commands/addContributor.ts

```diff
@@ -1,6 +1,6 @@
 import { SlashCommandBuilder } from '@discordjs/builders';
 import { CommandInteraction } from 'discord.js';
-import { isContributor } from '../utils/airTableCalls';
+import { isContributor, createContributor } from '../utils/index';
```

Comment:
js automatically selects the `index` file when referring to folders, so you can just point imports to a folder.

```suggestion
import { isContributor, createContributor } from '../utils';
```

Reply:
TIL!